build.gradle

```diff
@@ -96,7 +96,7 @@ buildScan {
 
 ext {
     daggerVersion = '2.22.1'
-    fluxCVersion = 'c5ea220f70c3a77681d376a17ce5c57e26c1c392'
+    fluxCVersion = '1.6.2'
```

Comment:
Looks like the 1.6.2 tag is pointing to https://github.com/wordpress-mobile/WordPress-FluxC-Android/commit/a38f48b532d3f2e3605d5024bce04f1149e3c52c, a commit quite older than the hash it replaces (`c5ea220f70c3a77681d376a17ce5c57e26c1c392`).

Can you elaborate on this @mchowning ? Thanks!

Reply:
>Looks like the 1.6.2 tag is pointing to wordpress-mobile/WordPress-FluxC-Android@a38f48b, a commit quite older than the hash it replaces (c5ea220f70c3a77681d376a17ce5c57e26c1c392).

I believe [1.6.2/a38f48b](https://github.com/wordpress-mobile/WordPress-FluxC-Android/commit/a38f48b532d3f2e3605d5024bce04f1149e3c52c) is actually newer than the original commit (`c5ea220f70c3a77681d376a17ce5c57e26c1c392`). Let me know if I'm missing something.

https://github.com/wordpress-mobile/WordPress-FluxC-Android/commits/a38f48b532d3f2e3605d5024bce04f1149e3c52c
![Screen Shot 2020-02-07 at 10 47 22 AM](https://user-images.githubusercontent.com/4656348/74043590-ab3ccf80-4997-11ea-9c0e-0d30120e0507.png)

Reply:
I believe [1.6.2/a38f48b](https://github.com/wordpress-mobile/WordPress-FluxC-Android/commit/a38f48b532d3f2e3605d5024bce04f1149e3c52c) is actually newer than the original commit (`c5ea220f70c3a77681d376a17ce5c57e26c1c392`). Let me know if I'm missing something.

https://github.com/wordpress-mobile/WordPress-FluxC-Android/commits/a38f48b532d3f2e3605d5024bce04f1149e3c52c
![Screen Shot 2020-02-07 at 10 47 22 AM](https://user-images.githubusercontent.com/4656348/74043590-ab3ccf80-4997-11ea-9c0e-0d30120e0507.png)

Reply:
Ah, you are right! Thanks for double checking!